packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -358,3 +359,32 @@ function createNativeOrder(fill: CollapsedFill): OptimizedMarketOrder {
         ...(fill as NativeCollapsedFill).nativeOrder,
     };
 }
+
+export function createSignedOrdersFromRfqtIndicativeQuotes(
+    quotes: RfqtIndicativeQuoteResponse[],
+): SignedOrderWithFillableAmounts[] {
+    return quotes.map(quote => {
+        return {
+            fillableMakerAssetAmount: quote.makerAssetAmount,
+            fillableTakerAssetAmount: quote.takerAssetAmount,
+            makerAssetAmount: quote.makerAssetAmount,
+            takerAssetAmount: quote.takerAssetAmount,
+            makerAssetData: quote.makerAssetData,
+            takerAssetData: quote.takerAssetData,
+            takerAddress: NULL_ADDRESS,
+            makerAddress: NULL_ADDRESS,
+            senderAddress: NULL_ADDRESS,
+            feeRecipientAddress: NULL_ADDRESS,
+            salt: ZERO_AMOUNT, // generatePseudoRandomSalt(),
+            expirationTimeSeconds: new BigNumber(Math.floor(Date.now() / ONE_SECOND_MS) + ONE_HOUR_IN_SECONDS),
```

Comment:
In a future revision I def think we should expect an expirationTimeSeconds from the MMer and use that here

Reply:
I put it at zero for now: https://github.com/0xProject/0x-monorepo/pull/2555/commits/ad7868ebe10b902c04f34cd2cfdac64b72dfb8bc